app/assets/javascripts/modules/improve-this-page.js

```diff
@@ -0,0 +1,160 @@
+(function (Modules) {
+  "use strict";
+
+  function ImproveThisPage () {
+    this.controller = null;
+    this.view = null;
+
+    this.start = function ($element) {
+      this.view = new View($element);
+      this.controller = new Controller(this.view);
+
+      this.controller.init();
+    }
+  };
+
+  function View ($element) {
+    var that = this;
+
+    this.pageIsUsefulButton = $element.find('.js-page-is-useful');
+    this.offerFeedbackButton = $element.find('.js-offer-feedback');
+    this.feedbackForm = $element.find('.js-feedback-form');
+    this.submitFeedbackButton = that.feedbackForm.find('[type=submit]');
+    this.prompt = $element.find('.js-prompt');
+
+    this.onPageIsUsefulButtonClicked = function (callback) {
```

Comment:
I've tried to keep the jQuery interacting with the DOM bits separate from the orchestration bit.

It's the idea behind the segregated DOM which I think is cool. http://radar.oreilly.com/2014/01/keeping-jquery-in-check.html http://martinfowler.com/bliki/SegregatedDOM.html